src/DataProcessor.ts

```diff
@@ -148,6 +149,9 @@ export class DataProcessor {
     }
     let mEvent: MaintenanceEvent
     switch (payload.task) {
+      case 'syncdeviceinfo':
+        mEvent = { type: SyncDeviceInfoEventType, clientId, payload }
+        break
```

Comment:
and then this line would become
deviceInfo: payload as DeviceInfo

Reply:
fixed